pkg/format/format.go

```diff
@@ -55,3 +51,18 @@ func info(podName string, containerName string, l string) {
 		log.Print("Pod Update", field.M{"Pod": podName, "Container": containerName, "Out": l})
 	}
 }
+
+func LogWithCtx(ctx context.Context, podName string, containerName string, output string) {
+	if output != "" {
+		logs := regexp.MustCompile("[\r\n]").Split(output, -1)
```

Comment:
Perform the reg exp compilation only once by initializing it in the package and then reusing it on all calls to `LogWithCtx`